goembed.go

```diff
@@ -166,6 +166,7 @@ func init() {
 		panic(err)
 	}
 	{{ $n }}, err = ioutil.ReadAll(r)
+	{{ $.VarName }}[{{ printf "%#v" $var }}] = {{ $n }}
```

Comment:
why does it need printf here?

Reply:
For %#v. We could also use "{{ $var }}", but letting Go do the formatting will cover all cases.

Reply:
Well, I understood that you need `printf` for `%#v`, but using `{{ $var }}` more directly seems fine. It's what the rest of the code around here does.

Reply:
The only other place in the code that I can find which refers to the map keys uses %#v, too: https://github.com/dsymonds/goembed/blob/b163b940fec5b808fc7157b3fb6b41e46febc85e/goembed.go#L42

Reply:
That was you, in 99ca9731. Not sure why you did that then, either.  ;-)

Just use `{{ $var }}` here. It's a supplied filename string, not any fancy data structure.

Reply:
Done.